commands/changed/__tests__/changed-command.test.js

```diff
@@ -41,6 +44,26 @@ const setupGitChanges = async (cwd, filePaths) => {
   await gitCommit(cwd, "Commit");
 };
 
+const setupGitChangesWithBranch = async (cwd, masterPaths, branchPaths) => {
```

Comment:
This helper and the tests should be moved to a focused test file under the version command, as described previously. `lerna changed` is just a lightweight wrapper around the internals used by `lerna version`, it doesn't have any logic itself beyond the list output options.